### PR TITLE
CONTRIBUTING.md created

### PR DESCRIPTION
**Pull request by code-reaper08**
+ Added a CONTRIBUTING.md file with the desired format like [this](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md).
+ Hope it closes this [issue](https://github.com/Code2Gether-Discord/JokesOnYou/issues/21).

Need any change or improvements just let me know.
I'm happy to help 😁✨

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+1. Fork the repo and create your branch from `main`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Issue that pull request!
+
+## We follow the "fork-and-pull" Git workflow
+
```
Comment (chrisK00) [inline CONTRIBUTING.md:29]: This looks awesome man! Could we also bring up that we prefer if they become a part of our discord so we can easily communicate and help each other out?

Comment (code-reaper08) [reply CONTRIBUTING.md:29]: well it looks fine but I don't track discord often tbh 😂.
I'll try to be active.

Thanks for asking

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+- Becoming a maintainer
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## Pull Requests
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `main`.
```
Comment (HellHunterMax) [inline CONTRIBUTING.md:21]: This should be dev branch not main.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Issue that pull request!
+
+## We follow the "fork-and-pull" Git workflow
+
+1. Fork the repository to your own Github account
+2. Clone the project to your machine
+3. Create a branch locally with a succinct but descriptive name
```
Comment (HellHunterMax) [inline CONTRIBUTING.md:32]: We have info in the wiki about naming your branch: 
[How to backlog](https://github.com/Code2Gether-Discord/JokesOnYou/wiki/How-To-Backlog)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+1. Fork the repo and create your branch from `main`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Issue that pull request!
+
+## We follow the "fork-and-pull" Git workflow
+
+1. Fork the repository to your own Github account
```
Comment (HellHunterMax) [inline CONTRIBUTING.md:30]: You can clone from the project itself. no need to fork it.
In dev you will find updated version of README which might help with this file.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+2. Clone the project to your machine
+3. Create a branch locally with a succinct but descriptive name
+4. Commit changes to the branch
+5. Following any formatting and testing guidelines specific to this repo
+6. Push changes to your fork
+7. Open a PR in our repository and follow the PR template so that we can efficiently review the changes.
```
Comment (HellHunterMax) [inline CONTRIBUTING.md:36]: Needs a fetch and pull and then merge dev into the new branch first so you know its most up to date and does not revert any changes made.

**Review by HellHunterMax — CHANGES_REQUESTED** [review]
I agree discussing and being active on Discord would be advisable.

**Comment by code-reaper08** [comment]
> I agree discussing and being active on Discord would be advisable.
> Check out dev branch to see the most up to date readme.

Hey thanks for pointing out 😊
I'll make those changes.

And about discord I'll try to be as active as much as possible.😄👍

**Pull request closed without merging**